dom/attr.ts

```diff
@@ -46,139 +44,8 @@ export function setAttrs (element : Element, attributes : OptionalKeyMap) : void
 
             element.setAttribute(
                 key,
-                (value === true) ? key : ("" + value)
+                (value === true) ? key : ("" + value),
             );
         }
     }
 }
-
-
-/**
- * Returns the attribute value for the given html node
- *
- * @deprecated use `element.getAttribute()` directly instead. Will be removed in v3.0
- */
-export function getAttr (element : Element, attribute : string) : null|string
```

Comment:
Can we get an entry in the `changelog.md` (or even an `update.md`) about the replacement for the now-missing functions?

Reply:
This will come at the end, as well as all the deprecations for 3.x